pyproject.toml

```diff
@@ -26,10 +26,11 @@ ilcdtool = { callable = "ilcdlib:cli.entrypoint" }
 python = "^3.11"
 urllib3 = { version = ">=1.26.17,<2.0.0" }
 requests = { version = ">=2.1.0,<3.0.0" }
-openepd = { version = ">=7.0.0" }
+openepd = { git = "https://github.com/cchangelabs/openepd.git", branch = "fix/openepd-model-declarations" }
```

Comment:
Don't forget to revert before merge.

Reply:
I won't; this will changed after releasing of openepd